haskell-language-server.cabal

```diff
@@ -1,7 +1,7 @@
 cabal-version:      3.0
 category:           Development
 name:               haskell-language-server
-version:            2.5.0.0
+version:            2.5.0.1
```

Comment:
HLS and all its plugin use lockstep versioning. Thus, all versions get universally bumped on release and you shouldn't bump versions in PRs, as this throws off our automation.
Just keep it at `2.5.0.0` :)